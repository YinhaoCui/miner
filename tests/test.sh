#!/bin/bash

do_diff() {
  # $1 expected
  # $2 is produced
  # $3 Test name

  echo -n "$3 test "
  diff $1 $2 > /dev/null
  if [ $? != 0 ]; then
    echo FAILED
    echo ======
    echo Expected Output:
    echo ++++++++++++++++
    cat $1
    echo =====================
    echo Actual Output:
    echo ++++++++++++++++
    cat $2
    exit 1
  else
    echo PASSED
  fi
}


echo ======================================================
echo ====================== TEST $1 ========================
echo ======================================================
ACTUAL=tests/test.$1.out
EXPECT=tests/test.$1.expected

# ./miner/miner < tests/test.$1.in > $ACTUAL
timeout 10 ./miner/miner < tests/test.$1.in > $ACTUAL
if [ $? == 0 ]; then 
  grep -v Thread $ACTUAL > tests/test.$1.basic
  grep -v Thread $EXPECT > tests/test.$1.exp_bsc
  do_diff tests/test.$1.exp_bsc tests/test.$1.basic "Basic functionality"

  grep Thread $ACTUAL | sed 's/ checking.*$//' | sort -u > tests/test.$1.thread
  grep Thread $EXPECT | sed 's/ checking.*$//' | sort -u > tests/test.$1.exp_thd
  do_diff tests/test.$1.exp_thd tests/test.$1.thread "Thread number"

  if grep 'Thread 1' tests/test.$1.out > /dev/null; then
    if python3 tests/thread_check.py < $ACTUAL; then
      echo Thread execution ordering test PASSED
    else
      echo Thread execution ordering test FAILED
      exit 1
    fi
  else
    grep Thread $ACTUAL > tests/test.$1.trd
    grep Thread $EXPECT > tests/test.$1.exp_trd
    do_diff tests/test.$1.exp_trd tests/test.$1.trd "Thread execution ordering"
  fi
elif [ $? == 124 ]; then
  echo TIMEOUT
  exit 1
else 
  echo Abnormal program termination: the program crashed
  echo Exit code $?
  exit 1
fi
