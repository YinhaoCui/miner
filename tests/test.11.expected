Adding transaction: 12358 Cat Dov 150000 200
Block mined: 1 0 0x00000000 1
12358 Cat Dov 150000 200
Thread 0 checking nonce 0x00000000
Thread 0 checking nonce 0x00000010
Thread 0 checking nonce 0x00000020
Thread 0 checking nonce 0x00000030
Thread 0 checking nonce 0x00000040
Thread 0 checking nonce 0x00000050
Thread 0 checking nonce 0x00000060
Thread 0 checking nonce 0x00000070
Thread 0 checking nonce 0x00000080
Thread 0 checking nonce 0x00000090
Thread 0 checking nonce 0x000000a0
Thread 0 checking nonce 0x000000b0
Thread 0 checking nonce 0x000000c0
Thread 0 checking nonce 0x000000d0
Thread 0 checking nonce 0x000000e0
Thread 0 checking nonce 0x000000f0
Thread 0 checking nonce 0x00000100
Thread 0 checking nonce 0x00000110
Thread 0 checking nonce 0x00000120
Thread 0 checking nonce 0x00000130
Thread 1 checking nonce 0x00000001
Thread 4 checking nonce 0x00000004
Thread 1 checking nonce 0x00000011
Thread 1 checking nonce 0x00000021
Thread 1 checking nonce 0x00000031
Thread 1 checking nonce 0x00000041
Thread 0 checking nonce 0x00000140
Thread 1 checking nonce 0x00000051
Thread 4 checking nonce 0x00000014
Thread 4 checking nonce 0x00000024
Thread 3 checking nonce 0x00000003
Thread 7 checking nonce 0x00000007
Thread 6 checking nonce 0x00000006
Thread 2 checking nonce 0x00000002
Thread 8 checking nonce 0x00000008
Thread 2 checking nonce 0x00000012
Thread 8 checking nonce 0x00000018
Thread 2 checking nonce 0x00000022
Thread 2 checking nonce 0x00000032
Thread 5 checking nonce 0x00000005
Thread 5 checking nonce 0x00000015
Thread 8 checking nonce 0x00000028
Thread 4 checking nonce 0x00000034
Thread 8 checking nonce 0x00000038
Thread 5 checking nonce 0x00000025
Thread 0 checking nonce 0x00000150
Thread 8 checking nonce 0x00000048
Thread 6 checking nonce 0x00000016
Thread 0 checking nonce 0x00000160
Thread 7 checking nonce 0x00000017
Thread 11 checking nonce 0x0000000b
Thread 7 checking nonce 0x00000027
Thread 11 checking nonce 0x0000001b
Thread 4 checking nonce 0x00000044
Thread 11 checking nonce 0x0000002b
Thread 2 checking nonce 0x00000042
Thread 1 checking nonce 0x00000061
Thread 13 checking nonce 0x0000000d
Thread 14 checking nonce 0x0000000e
Thread 15 checking nonce 0x0000000f
Thread 5 checking nonce 0x00000035
Thread 5 checking nonce 0x00000045
Thread 5 checking nonce 0x00000055
Thread 12 checking nonce 0x0000000c
Thread 4 checking nonce 0x00000054
Thread 10 checking nonce 0x0000000a
Thread 4 checking nonce 0x00000064
Thread 10 checking nonce 0x0000001a
Thread 4 checking nonce 0x00000074
Thread 10 checking nonce 0x0000002a
Thread 4 checking nonce 0x00000084
Thread 10 checking nonce 0x0000003a
Thread 4 checking nonce 0x00000094
Thread 1 checking nonce 0x00000071
Thread 11 checking nonce 0x0000003b
Thread 1 checking nonce 0x00000081
Thread 4 checking nonce 0x000000a4
Thread 1 checking nonce 0x00000091
Thread 4 checking nonce 0x000000b4
Thread 10 checking nonce 0x0000004a
Thread 10 checking nonce 0x0000005a
Thread 10 checking nonce 0x0000006a
Thread 9 checking nonce 0x00000009
Thread 10 checking nonce 0x0000007a
Thread 9 checking nonce 0x00000019
Thread 10 checking nonce 0x0000008a
Thread 2 checking nonce 0x00000052
Thread 9 checking nonce 0x00000029
Thread 10 checking nonce 0x0000009a
Thread 10 checking nonce 0x000000aa
Thread 1 checking nonce 0x000000a1
Thread 2 checking nonce 0x00000062
Thread 9 checking nonce 0x00000039
Thread 2 checking nonce 0x00000072
Thread 11 checking nonce 0x0000004b
Thread 4 checking nonce 0x000000c4
Thread 1 checking nonce 0x000000b1
Thread 0 checking nonce 0x00000170
Thread 1 checking nonce 0x000000c1
Thread 0 checking nonce 0x00000180
Thread 1 checking nonce 0x000000d1
Thread 7 checking nonce 0x00000037
Thread 1 checking nonce 0x000000e1
Thread 2 checking nonce 0x00000082
Thread 7 checking nonce 0x00000047
Thread 0 checking nonce 0x00000190
Thread 7 checking nonce 0x00000057
Thread 7 checking nonce 0x00000067
Thread 2 checking nonce 0x00000092
Thread 8 checking nonce 0x00000058
Thread 0 checking nonce 0x000001a0
Thread 4 checking nonce 0x000000d4
Thread 12 checking nonce 0x0000001c
Thread 3 checking nonce 0x00000013
Thread 3 checking nonce 0x00000023
Thread 3 checking nonce 0x00000033
Thread 1 checking nonce 0x000000f1
Thread 3 checking nonce 0x00000043
Thread 1 checking nonce 0x00000101
Thread 3 checking nonce 0x00000053
Thread 1 checking nonce 0x00000111
Thread 9 checking nonce 0x00000049
Thread 3 checking nonce 0x00000063
Thread 11 checking nonce 0x0000005b
Thread 3 checking nonce 0x00000073
Thread 3 checking nonce 0x00000083
Thread 1 checking nonce 0x00000121
Thread 3 checking nonce 0x00000093
Thread 1 checking nonce 0x00000131
Thread 3 checking nonce 0x000000a3
Thread 4 checking nonce 0x000000e4
Thread 3 checking nonce 0x000000b3
Thread 10 checking nonce 0x000000ba
Thread 0 checking nonce 0x000001b0
Thread 8 checking nonce 0x00000068
Thread 0 checking nonce 0x000001c0
Thread 5 checking nonce 0x00000065
Thread 0 checking nonce 0x000001d0
Thread 0 checking nonce 0x000001e0
Thread 15 checking nonce 0x0000001f
Thread 0 checking nonce 0x000001f0
Thread 15 checking nonce 0x0000002f
Thread 6 checking nonce 0x00000026
Thread 14 checking nonce 0x0000001e
Thread 6 checking nonce 0x00000036
Thread 4 checking nonce 0x000000f4
Thread 14 checking nonce 0x0000002e
Thread 1 checking nonce 0x00000141
Thread 4 checking nonce 0x00000104
Thread 9 checking nonce 0x00000059
Thread 4 checking nonce 0x00000114
Thread 4 checking nonce 0x00000124
Thread 8 checking nonce 0x00000078
Thread 9 checking nonce 0x00000069
Thread 2 checking nonce 0x000000a2
Thread 12 checking nonce 0x0000002c
Thread 9 checking nonce 0x00000079
Thread 2 checking nonce 0x000000b2
Thread 2 checking nonce 0x000000c2
Thread 5 checking nonce 0x00000075
Thread 2 checking nonce 0x000000d2
Thread 7 checking nonce 0x00000077
Thread 2 checking nonce 0x000000e2
Thread 9 checking nonce 0x00000089
Thread 5 checking nonce 0x00000085
Thread 7 checking nonce 0x00000087
Thread 5 checking nonce 0x00000095
Thread 4 checking nonce 0x00000134
Thread 4 checking nonce 0x00000144
Thread 5 checking nonce 0x000000a5
Thread 4 checking nonce 0x00000154
Thread 9 checking nonce 0x00000099
Thread 14 checking nonce 0x0000003e
Thread 4 checking nonce 0x00000164
Thread 14 checking nonce 0x0000004e
Thread 4 checking nonce 0x00000174
Thread 9 checking nonce 0x000000a9
Thread 13 checking nonce 0x0000001d
Thread 15 checking nonce 0x0000003f
Thread 2 checking nonce 0x000000f2
Thread 15 checking nonce 0x0000004f
Thread 2 checking nonce 0x00000102
Thread 15 checking nonce 0x0000005f
Thread 2 checking nonce 0x00000112
Thread 15 checking nonce 0x0000006f
Thread 3 checking nonce 0x000000c3
Thread 1 checking nonce 0x00000151
Thread 3 checking nonce 0x000000d3
Thread 5 checking nonce 0x000000b5
Thread 3 checking nonce 0x000000e3
Thread 5 checking nonce 0x000000c5
Thread 1 checking nonce 0x00000161
Thread 1 checking nonce 0x00000171
Thread 3 checking nonce 0x000000f3
Thread 14 checking nonce 0x0000005e
Thread 3 checking nonce 0x00000103
Thread 8 checking nonce 0x00000088
Thread 3 checking nonce 0x00000113
Thread 10 checking nonce 0x000000ca
Thread 8 checking nonce 0x00000098
Thread 10 checking nonce 0x000000da
Thread 4 checking nonce 0x00000184
Thread 3 checking nonce 0x00000123
Thread 0 checking nonce 0x00000200
Thread 4 checking nonce 0x00000194
Thread 4 checking nonce 0x000001a4
Thread 3 checking nonce 0x00000133
Thread 8 checking nonce 0x000000a8
Thread 11 checking nonce 0x0000006b
Thread 7 checking nonce 0x00000097
Thread 10 checking nonce 0x000000ea
Thread 2 checking nonce 0x00000122
Thread 2 checking nonce 0x00000132
Thread 12 checking nonce 0x0000003c
Thread 12 checking nonce 0x0000004c
Thread 12 checking nonce 0x0000005c
Thread 12 checking nonce 0x0000006c
Thread 12 checking nonce 0x0000007c
Thread 12 checking nonce 0x0000008c
Thread 12 checking nonce 0x0000009c
Thread 12 checking nonce 0x000000ac
Thread 12 checking nonce 0x000000bc
Thread 2 checking nonce 0x00000142
Thread 12 checking nonce 0x000000cc
Thread 8 checking nonce 0x000000b8
Thread 12 checking nonce 0x000000dc
Thread 10 checking nonce 0x000000fa
Thread 11 checking nonce 0x0000007b
Thread 8 checking nonce 0x000000c8
Thread 11 checking nonce 0x0000008b
Thread 11 checking nonce 0x0000009b
Thread 11 checking nonce 0x000000ab
Thread 6 checking nonce 0x00000046
Thread 11 checking nonce 0x000000bb
Thread 6 checking nonce 0x00000056
Thread 10 checking nonce 0x0000010a
Thread 11 checking nonce 0x000000cb
Thread 10 checking nonce 0x0000011a
Thread 6 checking nonce 0x00000066
Thread 10 checking nonce 0x0000012a
Thread 6 checking nonce 0x00000076
Thread 10 checking nonce 0x0000013a
Thread 11 checking nonce 0x000000db
Thread 6 checking nonce 0x00000086
Thread 10 checking nonce 0x0000014a
Thread 11 checking nonce 0x000000eb
Thread 6 checking nonce 0x00000096
Thread 10 checking nonce 0x0000015a
Thread 6 checking nonce 0x000000a6
Thread 11 checking nonce 0x000000fb
Thread 4 checking nonce 0x000001b4
Thread 11 checking nonce 0x0000010b
Thread 10 checking nonce 0x0000016a
Thread 4 checking nonce 0x000001c4
Thread 11 checking nonce 0x0000011b
Thread 10 checking nonce 0x0000017a
Thread 4 checking nonce 0x000001d4
Thread 11 checking nonce 0x0000012b
Thread 14 checking nonce 0x0000006e
Thread 14 checking nonce 0x0000007e
Thread 14 checking nonce 0x0000008e
Thread 14 checking nonce 0x0000009e
Thread 14 checking nonce 0x000000ae
Thread 14 checking nonce 0x000000be
Thread 7 checking nonce 0x000000a7
Thread 3 checking nonce 0x00000143
Thread 9 checking nonce 0x000000b9
Thread 9 checking nonce 0x000000c9
Thread 3 checking nonce 0x00000153
Thread 4 checking nonce 0x000001e4
Thread 4 checking nonce 0x000001f4
Thread 4 checking nonce 0x00000204
Thread 2 checking nonce 0x00000152
Thread 4 checking nonce 0x00000214
Thread 15 checking nonce 0x0000007f
Thread 2 checking nonce 0x00000162
Thread 4 checking nonce 0x00000224
Thread 2 checking nonce 0x00000172
Thread 4 checking nonce 0x00000234
Thread 15 checking nonce 0x0000008f
Thread 2 checking nonce 0x00000182
Thread 13 checking nonce 0x0000002d
Thread 2 checking nonce 0x00000192
Thread 5 checking nonce 0x000000d5
Thread 2 checking nonce 0x000001a2
Thread 1 checking nonce 0x00000181
Thread 10 checking nonce 0x0000018a
Thread 1 checking nonce 0x00000191
Thread 10 checking nonce 0x0000019a
Thread 3 checking nonce 0x00000163
Thread 1 checking nonce 0x000001a1
Thread 0 checking nonce 0x00000210
Thread 10 checking nonce 0x000001aa
Thread 1 checking nonce 0x000001b1
Thread 10 checking nonce 0x000001ba
Thread 0 checking nonce 0x00000220
Thread 10 checking nonce 0x000001ca
Thread 8 checking nonce 0x000000d8
Thread 10 checking nonce 0x000001da
Thread 10 checking nonce 0x000001ea
Thread 10 checking nonce 0x000001fa
Thread 10 checking nonce 0x0000020a
Thread 10 checking nonce 0x0000021a
Thread 10 checking nonce 0x0000022a
Thread 10 checking nonce 0x0000023a
Thread 10 checking nonce 0x0000024a
Thread 8 checking nonce 0x000000e8
Thread 0 checking nonce 0x00000230
Thread 0 checking nonce 0x00000240
Thread 8 checking nonce 0x000000f8
Thread 2 checking nonce 0x000001b2
Thread 7 checking nonce 0x000000b7
Thread 8 checking nonce 0x00000108
Thread 2 checking nonce 0x000001c2
Thread 0 checking nonce 0x00000250
Thread 7 checking nonce 0x000000c7
Thread 8 checking nonce 0x00000118
Thread 0 checking nonce 0x00000260
Thread 8 checking nonce 0x00000128
Thread 8 checking nonce 0x00000138
Thread 5 checking nonce 0x000000e5
Thread 14 checking nonce 0x000000ce
Thread 0 checking nonce 0x00000270
Thread 14 checking nonce 0x000000de
Thread 6 checking nonce 0x000000b6
Thread 13 checking nonce 0x0000003d
Thread 12 checking nonce 0x000000ec
Thread 14 checking nonce 0x000000ee
Thread 0 checking nonce 0x00000280
Thread 12 checking nonce 0x000000fc
Thread 0 checking nonce 0x00000290
Thread 14 checking nonce 0x000000fe
Thread 12 checking nonce 0x0000010c
Thread 6 checking nonce 0x000000c6
Thread 15 checking nonce 0x0000009f
Thread 9 checking nonce 0x000000d9
Thread 6 checking nonce 0x000000d6
Thread 9 checking nonce 0x000000e9
Thread 1 checking nonce 0x000001c1
Thread 6 checking nonce 0x000000e6
Thread 5 checking nonce 0x000000f5
Thread 3 checking nonce 0x00000173
Thread 6 checking nonce 0x000000f6
Thread 6 checking nonce 0x00000106
Thread 6 checking nonce 0x00000116
Thread 6 checking nonce 0x00000126
Thread 6 checking nonce 0x00000136
Thread 6 checking nonce 0x00000146
Thread 4 checking nonce 0x00000244
Thread 6 checking nonce 0x00000156
Thread 4 checking nonce 0x00000254
Thread 6 checking nonce 0x00000166
Thread 8 checking nonce 0x00000148
Thread 4 checking nonce 0x00000264
Thread 8 checking nonce 0x00000158
Thread 6 checking nonce 0x00000176
Thread 4 checking nonce 0x00000274
Thread 4 checking nonce 0x00000284
Thread 13 checking nonce 0x0000004d
Thread 8 checking nonce 0x00000168
Thread 4 checking nonce 0x00000294
Thread 3 checking nonce 0x00000183
Thread 14 checking nonce 0x0000010e
Thread 12 checking nonce 0x0000011c
Thread 14 checking nonce 0x0000011e
Thread 10 checking nonce 0x0000025a
Thread 12 checking nonce 0x0000012c
Thread 9 checking nonce 0x000000f9
Thread 10 checking nonce 0x0000026a
Thread 12 checking nonce 0x0000013c
Thread 9 checking nonce 0x00000109
Thread 12 checking nonce 0x0000014c
Thread 15 checking nonce 0x000000af
Thread 15 checking nonce 0x000000bf
Thread 1 checking nonce 0x000001d1
Thread 5 checking nonce 0x00000105
Thread 0 checking nonce 0x000002a0
Thread 1 checking nonce 0x000001e1
Thread 1 checking nonce 0x000001f1
Thread 5 checking nonce 0x00000115
Thread 8 checking nonce 0x00000178
Thread 8 checking nonce 0x00000188
Thread 5 checking nonce 0x00000125
Thread 7 checking nonce 0x000000d7
Thread 2 checking nonce 0x000001d2
Thread 5 checking nonce 0x00000135
Thread 1 checking nonce 0x00000201
Thread 4 checking nonce 0x000002a4
Thread 1 checking nonce 0x00000211
Thread 6 checking nonce 0x00000186
Thread 1 checking nonce 0x00000221
Thread 6 checking nonce 0x00000196
Thread 0 checking nonce 0x000002b0
Thread 10 checking nonce 0x0000027a
Thread 6 checking nonce 0x000001a6
Thread 10 checking nonce 0x0000028a
Thread 15 checking nonce 0x000000cf
Thread 6 checking nonce 0x000001b6
Thread 10 checking nonce 0x0000029a
Thread 6 checking nonce 0x000001c6
Thread 7 checking nonce 0x000000e7
Thread 10 checking nonce 0x000002aa
Thread 7 checking nonce 0x000000f7
Thread 11 checking nonce 0x0000013b
Thread 8 checking nonce 0x00000198
Thread 5 checking nonce 0x00000145
Thread 12 checking nonce 0x0000015c
Thread 5 checking nonce 0x00000155
Thread 5 checking nonce 0x00000165
Thread 2 checking nonce 0x000001e2
Thread 5 checking nonce 0x00000175
Thread 5 checking nonce 0x00000185
Thread 14 checking nonce 0x0000012e
Thread 5 checking nonce 0x00000195
Thread 14 checking nonce 0x0000013e
Thread 1 checking nonce 0x00000231
Thread 13 checking nonce 0x0000005d
Thread 1 checking nonce 0x00000241
Thread 13 checking nonce 0x0000006d
Thread 1 checking nonce 0x00000251
Thread 13 checking nonce 0x0000007d
Thread 15 checking nonce 0x000000df
Thread 1 checking nonce 0x00000261
Thread 13 checking nonce 0x0000008d
Thread 15 checking nonce 0x000000ef
Thread 13 checking nonce 0x0000009d
Thread 5 checking nonce 0x000001a5
Thread 5 checking nonce 0x000001b5
Thread 5 checking nonce 0x000001c5
Thread 7 checking nonce 0x00000107
Thread 7 checking nonce 0x00000117
Thread 8 checking nonce 0x000001a8
Thread 7 checking nonce 0x00000127
Thread 8 checking nonce 0x000001b8
Thread 7 checking nonce 0x00000137
Thread 14 checking nonce 0x0000014e
Thread 8 checking nonce 0x000001c8
Thread 14 checking nonce 0x0000015e
Thread 11 checking nonce 0x0000014b
Thread 14 checking nonce 0x0000016e
Thread 8 checking nonce 0x000001d8
Thread 8 checking nonce 0x000001e8
Thread 15 checking nonce 0x000000ff
Thread 5 checking nonce 0x000001d5
Thread 15 checking nonce 0x0000010f
Thread 6 checking nonce 0x000001d6
Thread 15 checking nonce 0x0000011f
Thread 13 checking nonce 0x000000ad
Thread 13 checking nonce 0x000000bd
Thread 3 checking nonce 0x00000193
Thread 13 checking nonce 0x000000cd
Thread 8 checking nonce 0x000001f8
Thread 9 checking nonce 0x00000119
Thread 13 checking nonce 0x000000dd
Thread 15 checking nonce 0x0000012f
Thread 2 checking nonce 0x000001f2
Thread 9 checking nonce 0x00000129
Thread 2 checking nonce 0x00000202
Thread 9 checking nonce 0x00000139
Thread 6 checking nonce 0x000001e6
Thread 9 checking nonce 0x00000149
Thread 9 checking nonce 0x00000159
Thread 9 checking nonce 0x00000169
Thread 9 checking nonce 0x00000179
Thread 11 checking nonce 0x0000015b
Thread 9 checking nonce 0x00000189
Thread 11 checking nonce 0x0000016b
Thread 9 checking nonce 0x00000199
Thread 7 checking nonce 0x00000147
Thread 11 checking nonce 0x0000017b
Thread 2 checking nonce 0x00000212
Thread 9 checking nonce 0x000001a9
Thread 11 checking nonce 0x0000018b
Thread 11 checking nonce 0x0000019b
Thread 2 checking nonce 0x00000222
Thread 7 checking nonce 0x00000157
Thread 11 checking nonce 0x000001ab
Thread 3 checking nonce 0x000001a3
Thread 14 checking nonce 0x0000017e
Thread 14 checking nonce 0x0000018e
Thread 6 checking nonce 0x000001f6
Thread 14 checking nonce 0x0000019e
Thread 14 checking nonce 0x000001ae
Thread 6 checking nonce 0x00000206
Thread 7 checking nonce 0x00000167
Thread 6 checking nonce 0x00000216
Thread 3 checking nonce 0x000001b3
Thread 7 checking nonce 0x00000177
Thread 6 checking nonce 0x00000226
Thread 7 checking nonce 0x00000187
Thread 1 checking nonce 0x00000271
Thread 6 checking nonce 0x00000236
Thread 6 checking nonce 0x00000246
Thread 1 checking nonce 0x00000281
Thread 6 checking nonce 0x00000256
Thread 6 checking nonce 0x00000266
Thread 3 checking nonce 0x000001c3
Thread 5 checking nonce 0x000001e5
Thread 8 checking nonce 0x00000208
Thread 7 checking nonce 0x00000197
Thread 12 checking nonce 0x0000016c
Thread 7 checking nonce 0x000001a7
Thread 14 checking nonce 0x000001be
Thread 7 checking nonce 0x000001b7
Thread 1 checking nonce 0x00000291
Thread 7 checking nonce 0x000001c7
Thread 1 checking nonce 0x000002a1
Thread 2 checking nonce 0x00000232
Thread 2 checking nonce 0x00000242
Thread 2 checking nonce 0x00000252
Thread 2 checking nonce 0x00000262
Thread 2 checking nonce 0x00000272
Thread 7 checking nonce 0x000001d7
Thread 1 checking nonce 0x000002b1
Thread 2 checking nonce 0x00000282
Thread 2 checking nonce 0x00000292
Thread 9 checking nonce 0x000001b9
Thread 6 checking nonce 0x00000276
Thread 13 checking nonce 0x000000ed
Thread 6 checking nonce 0x00000286
Thread 12 checking nonce 0x0000017c
Thread 13 checking nonce 0x000000fd
Thread 6 checking nonce 0x00000296
Thread 6 checking nonce 0x000002a6
Thread 12 checking nonce 0x0000018c
Thread 12 checking nonce 0x0000019c
Thread 12 checking nonce 0x000001ac
Thread 12 checking nonce 0x000001bc
Thread 12 checking nonce 0x000001cc
Thread 12 checking nonce 0x000001dc
Thread 5 checking nonce 0x000001f5
Thread 14 checking nonce 0x000001ce
Thread 3 checking nonce 0x000001d3
Thread 14 checking nonce 0x000001de
Thread 3 checking nonce 0x000001e3
Thread 14 checking nonce 0x000001ee
Thread 3 checking nonce 0x000001f3
Thread 13 checking nonce 0x0000010d
Thread 3 checking nonce 0x00000203
Thread 15 checking nonce 0x0000013f
Thread 13 checking nonce 0x0000011d
Thread 14 checking nonce 0x000001fe
Thread 11 checking nonce 0x000001bb
Thread 7 checking nonce 0x000001e7
Thread 11 checking nonce 0x000001cb
Thread 8 checking nonce 0x00000218
Thread 11 checking nonce 0x000001db
Thread 12 checking nonce 0x000001ec
Thread 8 checking nonce 0x00000228
Thread 14 checking nonce 0x0000020e
Thread 8 checking nonce 0x00000238
Thread 8 checking nonce 0x00000248
Thread 8 checking nonce 0x00000258
Thread 8 checking nonce 0x00000268
Thread 8 checking nonce 0x00000278
Thread 8 checking nonce 0x00000288
Thread 7 checking nonce 0x000001f7
Thread 14 checking nonce 0x0000021e
Thread 15 checking nonce 0x0000014f
Thread 9 checking nonce 0x000001c9
Thread 15 checking nonce 0x0000015f
Thread 14 checking nonce 0x0000022e
Thread 8 checking nonce 0x00000298
Thread 8 checking nonce 0x000002a8
Thread 11 checking nonce 0x000001eb
Thread 14 checking nonce 0x0000023e
Thread 14 checking nonce 0x0000024e
Thread 14 checking nonce 0x0000025e
Thread 9 checking nonce 0x000001d9
Thread 14 checking nonce 0x0000026e
Thread 9 checking nonce 0x000001e9
Thread 14 checking nonce 0x0000027e
Thread 9 checking nonce 0x000001f9
Thread 14 checking nonce 0x0000028e
Thread 9 checking nonce 0x00000209
Thread 11 checking nonce 0x000001fb
Thread 3 checking nonce 0x00000213
Thread 3 checking nonce 0x00000223
Thread 3 checking nonce 0x00000233
Thread 2 checking nonce 0x000002a2
Thread 3 checking nonce 0x00000243
Thread 2 checking nonce 0x000002b2
Thread 3 checking nonce 0x00000253
Thread 7 checking nonce 0x00000207
Thread 15 checking nonce 0x0000016f
Thread 11 checking nonce 0x0000020b
Thread 7 checking nonce 0x00000217
Thread 12 checking nonce 0x000001fc
Thread 15 checking nonce 0x0000017f
Thread 15 checking nonce 0x0000018f
Thread 15 checking nonce 0x0000019f
Thread 15 checking nonce 0x000001af
Thread 14 checking nonce 0x0000029e
Thread 15 checking nonce 0x000001bf
Thread 13 checking nonce 0x0000012d
Thread 14 checking nonce 0x000002ae
Thread 15 checking nonce 0x000001cf
Thread 12 checking nonce 0x0000020c
Thread 12 checking nonce 0x0000021c
Thread 12 checking nonce 0x0000022c
Thread 12 checking nonce 0x0000023c
Thread 15 checking nonce 0x000001df
Thread 12 checking nonce 0x0000024c
Thread 15 checking nonce 0x000001ef
Thread 12 checking nonce 0x0000025c
Thread 15 checking nonce 0x000001ff
Thread 12 checking nonce 0x0000026c
Thread 15 checking nonce 0x0000020f
Thread 15 checking nonce 0x0000021f
Thread 12 checking nonce 0x0000027c
Thread 15 checking nonce 0x0000022f
Thread 9 checking nonce 0x00000219
Thread 11 checking nonce 0x0000021b
Thread 5 checking nonce 0x00000205
Thread 5 checking nonce 0x00000215
Thread 7 checking nonce 0x00000227
Thread 5 checking nonce 0x00000225
Thread 7 checking nonce 0x00000237
Thread 12 checking nonce 0x0000028c
Thread 5 checking nonce 0x00000235
Thread 7 checking nonce 0x00000247
Thread 5 checking nonce 0x00000245
Thread 5 checking nonce 0x00000255
Thread 5 checking nonce 0x00000265
Thread 5 checking nonce 0x00000275
Thread 5 checking nonce 0x00000285
Thread 5 checking nonce 0x00000295
Thread 5 checking nonce 0x000002a5
Thread 11 checking nonce 0x0000022b
Thread 7 checking nonce 0x00000257
Thread 7 checking nonce 0x00000267
Thread 7 checking nonce 0x00000277
Thread 7 checking nonce 0x00000287
Thread 7 checking nonce 0x00000297
Thread 7 checking nonce 0x000002a7
Thread 12 checking nonce 0x0000029c
Thread 3 checking nonce 0x00000263
Thread 13 checking nonce 0x0000013d
Thread 3 checking nonce 0x00000273
Thread 9 checking nonce 0x00000229
Thread 9 checking nonce 0x00000239
Thread 9 checking nonce 0x00000249
Thread 3 checking nonce 0x00000283
Thread 9 checking nonce 0x00000259
Thread 11 checking nonce 0x0000023b
Thread 9 checking nonce 0x00000269
Thread 3 checking nonce 0x00000293
Thread 9 checking nonce 0x00000279
Thread 9 checking nonce 0x00000289
Thread 9 checking nonce 0x00000299
Thread 9 checking nonce 0x000002a9
Thread 3 checking nonce 0x000002a3
Thread 15 checking nonce 0x0000023f
Thread 12 checking nonce 0x000002ac
Thread 15 checking nonce 0x0000024f
Thread 15 checking nonce 0x0000025f
Thread 15 checking nonce 0x0000026f
Thread 11 checking nonce 0x0000024b
Thread 3 checking nonce 0x000002b3
Thread 11 checking nonce 0x0000025b
Thread 13 checking nonce 0x0000014d
Thread 13 checking nonce 0x0000015d
Thread 13 checking nonce 0x0000016d
Thread 15 checking nonce 0x0000027f
Thread 15 checking nonce 0x0000028f
Thread 15 checking nonce 0x0000029f
Thread 15 checking nonce 0x000002af
Thread 11 checking nonce 0x0000026b
Thread 11 checking nonce 0x0000027b
Thread 11 checking nonce 0x0000028b
Thread 11 checking nonce 0x0000029b
Thread 11 checking nonce 0x000002ab
Thread 13 checking nonce 0x0000017d
Thread 13 checking nonce 0x0000018d
Thread 13 checking nonce 0x0000019d
Thread 13 checking nonce 0x000001ad
Thread 13 checking nonce 0x000001bd
Thread 13 checking nonce 0x000001cd
Thread 13 checking nonce 0x000001dd
Thread 13 checking nonce 0x000001ed
Thread 13 checking nonce 0x000001fd
Thread 13 checking nonce 0x0000020d
Thread 13 checking nonce 0x0000021d
Thread 13 checking nonce 0x0000022d
Thread 13 checking nonce 0x0000023d
Thread 13 checking nonce 0x0000024d
Thread 13 checking nonce 0x0000025d
Thread 13 checking nonce 0x0000026d
Thread 13 checking nonce 0x0000027d
Thread 13 checking nonce 0x0000028d
Thread 13 checking nonce 0x0000029d
Thread 13 checking nonce 0x000002ad
0x000002a4 0x00b3f33c
